deploy/roles/full-app/templates/config/cometbft/genesis-mainnet.json

```diff
@@ -117,73 +119,10 @@
             "users": [
               {
                 "key": {
-                  "secp256k1": "Anj3t9k9qbWmLihDQYTRwzfCwo1M7SkXkyFatu6J1//4"
+                  "secp256k1": "A2Uj5mWM1u9AL+r7Q7ZYekezdiOXgx5YIJM1yJ1m1RPq"
                 },
-                "key_hash": "06E54A648823A1F12E1F03FED193C9FE0C030A65507FF09066BF9E067CD375D2",
+                "key_hash": "4682F5BE77499C54F63BDEF6734F8E388275342BFD845322D9377A5C312B7F8B",
                 "seed": 0
-              },
```

Comment:
I generated a new Secp256k1 key which will be used as the owner. All other accounts (`user1..9`) are deleted.